### PR TITLE
Add TrN for repository limit

### PR DESCRIPTION
**Pull request by evur**
Adds a different message for users who have a limit of zero (technically <1) repositories.

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -697,6 +697,7 @@ archive.issue.nocomment = This repo is archived. You cannot comment on issues.
 archive.pull.nocomment = This repo is archived. You cannot comment on pull requests.
 
 form.reach_limit_of_creation = You have already reached your limit of %d repositories.
```
Comment (a1012112796) [inline options/locale/locale_en-US.ini]: I think we need use ``TrN`` in here to handle singular and plural  differences.

Comment (evur) [reply options/locale/locale_en-US.ini]: What does "TrN" mean? I couldn't find anything.

Comment (a1012112796) [reply options/locale/locale_en-US.ini]: example:
https://github.com/go-gitea/gitea/blob/eb60a5d05478f56bf15220f49ed78f0f72af5698/templates/repo/activity.tmpl#L44

Comment (evur) [reply options/locale/locale_en-US.ini]: Thanks. Looks like this solution is better, I will try to implement it.

**Comment by EV3R4** [comment]
I wanted to work on the PR and first updated my fork... but GitHub automatically closed the PR. So please re-open it.

File: templates/repo/create.tmpl [hunk 2]
```diff
@@ -8,7 +8,11 @@
 					{{.i18n.Tr "new_repo"}}
 				</h3>
 				<div class="ui attached segment">
-					{{template "base/alert" .}}
```
Comment (lafriks) [inline templates/repo/create.tmpl:11]: This will break not to show other error messages

Comment (evur) [reply templates/repo/create.tmpl:11]: Are there other error messages?

Comment (lafriks) [reply templates/repo/create.tmpl:11]: If subbmiting form has any validation errors

**Comment by EV3R4** [comment]
Sorry, but I am not experienced enough with Go and the Gitea codebase. I thought I could make a little PR, but it turned out to be trickier.

File: templates/repo/create.tmpl [hunk 3]
```diff
@@ -8,7 +8,11 @@
...
+					{{if not .CanCreateRepo}}
```
Comment (6543) [inline templates/repo/create.tmpl:12]:
```suggestion
					{{template "base/alert" .}}
					{{if not .CanCreateRepo}}
```

**Review by 6543 — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Comment by 6543** [comment]
@EV3R4 could you apply this:
```diff
diff --git a/templates/repo/create.tmpl b/templates/repo/create.tmpl
index f6bffcffc..ad795387d 100644
--- a/templates/repo/create.tmpl
+++ b/templates/repo/create.tmpl
@@ -198,7 +198,7 @@
                                        <br/>
                                        <div class="inline field">
                                                <label></label>
-                                               <button class="ui green button">
+                                               <button class="ui green button{{if not .CanCreateRepo}} disabled{{end}}">
                                                        {{.i18n.Tr "repo.create_repo"}}
                                                </button>
                                                <a class="ui button" href="{{AppSubUrl}}/">{{.i18n.Tr "cancel"}}</a>
```

**Review by 6543 — CHANGES_REQUESTED** [review]
:point_up:

**Comment by zeripath** [comment]
@6543 it might be quicker to just do:

```
git fetch git@github.com:EV3R4/gitea.git master:pr-12492
git checkout pr-12492
# make your changes
# git commit
git push git@github.com:EV3R4/gitea.git pr-12492:master
```

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**